elixir/apps/domain/lib/domain/version.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Domain.Version do
   def fetch_version(user_agent) when is_binary(user_agent) do
     user_agent
+    |> URI.decode()
```

Comment:
@thomaseizinger this will decode the user_agent so the remaining parsing here works as intended.

Reply:
In fact, I can do this in a higher up module so that it's encoded as soon as it comes in.